### PR TITLE
Create a new package for UI / Visualization classes.

### PR DESCRIPTION
**Pull request by hernanmd**
Update BaselineOf

### CODE REVIEW

File: src/BaselineOfMatrixBenchmark/BaselineOfMatrixBenchmark.class.st [hunk 1]
```diff
@@ -26,13 +26,16 @@ BaselineOfMatrixBenchmark >> baseline: spec [
 				with: [ spec repository: 'github://ObjectProfile/Roassal3/src' ].
 
 			spec 
-				package: 'Matrix-Benchmarks' with: [ spec requires: #('AILinearAlgebra' 'PolyMath' 'CalipeLSBenchmark' 'Roassal3') ];
-				package: 'Matrix-Benchmarks-Tests' with: [ spec requires: #('Matrix-Benchmarks') ].
+				package: 'Matrix-Benchmarks' 			with: [ spec requires: #('AILinearAlgebra' 'PolyMath' 'CalipeLSBenchmark') ];
```
Comment (rakki-18) [inline src/BaselineOfMatrixBenchmark/BaselineOfMatrixBenchmark.class.st:29]: @hernanmd I think `Matrix-Benchmarks` should be replaced by `Matrix-Benchmarks-Core` throughout the baseline.

**Comment by rakki-18** [comment]
@hernanmd I saw that `MBVisualizer`class has been removed from `Matrix-Benchmarks-Core`. But I don't think it has been added in the `Matrix-Benchmarks-UI` package.

File: src/BaselineOfMatrixBenchmark/BaselineOfMatrixBenchmark.class.st (hunk 1 above)
Comment (hernanmd) [reply src/BaselineOfMatrixBenchmark/BaselineOfMatrixBenchmark.class.st:29]: Hi @rakki-18 no, the "Core" is just an internal package tag. The Baseline method spec should work without specifying the "Core" tag.

**Comment by hernanmd** [comment]
Done

File: src/BaselineOfMatrixBenchmark/BaselineOfMatrixBenchmark.class.st (hunk 1 above)
Comment (rakki-18) [reply src/BaselineOfMatrixBenchmark/BaselineOfMatrixBenchmark.class.st:29]: Oh right, my bad.

**Pull request merged**